zetaclient/zetacore/client_monitor.go

```diff
@@ -156,6 +157,7 @@ func (c *Client) monitorVoteInboundResult(
 		logger.Error().Str(logs.FieldZetaTx, zetaTxHash).Msg("failed to execute inbound vote")
 
 	case strings.Contains(txResult.RawLog, "out of gas"):
+		metrics.InboundVotesWithOutOfGasErrorsTotal.WithLabelValues(c.chain.Name).Inc()
```

Comment:
For more accuracy, this metrics update can be moved inside the `c.addReadyToExecuteInboundBallot` below, because we have a logic that enforces uniqueness of the out-of-gas vote.